iotdb-influxdb/src/test/java/org/apache/iotdb/influxdb/IoTDBInfluxDBTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.influxdb;
+
+import org.apache.iotdb.rpc.IoTDBConnectionException;
+import org.apache.iotdb.rpc.StatementExecutionException;
+import org.apache.iotdb.session.Session;
+import org.apache.iotdb.session.SessionDataSet;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+
+import org.junit.Before;
+
+import java.util.*;
+
+public class IoTDBInfluxDBTest {
+  private IoTDBInfluxDB iotDBInfluxDB;
+
+  private static final String LOCAL_HOST = "127.0.0.1";
+  private static Session session;
+  private static final String ROOT_T1 = "root.teststress.test1";
+  private static final String ROOT_T2 = "root.teststress.test2";
+
+  @Before
+  public void setUp() throws IoTDBConnectionException {
+    // 创建 连接
```

Comment:
```suggestion
    // create session
```

Reply:
I have changed all the Chinese in the project into English  :D :D